info-compiler/src/test/java/com/google/sps/InfoCompilerTest.java

```diff
@@ -201,20 +202,20 @@ public void storeElectionContestInDatabase_checkDatastoreEntityConstructionFromJ
     Entity electionEntity = datastore.run(electionQuery).next();
     List<Value<String>> candidatePositions =
         new ArrayList<>(electionEntity.getList("candidatePositions"));
-    assertThat(1).isEqualTo(candidatePositions.size());
-    assertThat(Arrays.asList(StringValue.newBuilder(
-                                 SINGLE_CONTEST_JSON.get("office").getAsString()).build()))
-        .isEqualTo(candidatePositions);
+    assertThat(candidatePositions.size()).isEqualTo(1);
+    assertThat(candidatePositions)
+        .isEqualTo(Arrays.asList(StringValue.newBuilder(
+                                 SINGLE_CONTEST_JSON.get("office").getAsString()).build()));
```

Comment:
nit: Use IterableSubject methods:
https://truth.dev/api/1.0.1/com/google/common/truth/IterableSubject.html

```suggestion
    assertThat(candidatePositions).hasSize(1);
    assertThat(candidatePositions)
        .containsExactly(StringValue.newBuilder(
                                 SINGLE_CONTEST_JSON.get("office").getAsString()).build());
```